device-catalog.html

```diff
@@ -9,186 +9,224 @@
-                            </li>
-                            <li>
-                                <input type="checkbox" name="blue" id="filter-color-blue" checked>
-                                <label for="filter-color-blue">Синий</label>
-                            </li>
-                            <li>
-                                <input type="checkbox" name="red" id="filter-color-red" checked>
-                                <label for="filter-color-red">Красный</label>
-                            </li>
-                            <li>
-                                <input type="checkbox" name="pink" id="filter-color-pink">
-                                <label for="filter-color-pink">Розовый</label>
-                            </li>
-                        </ul>
-                    </fieldset>
-                    <fieldset>
-                        <legend>Bluetooth</legend>
-                        <ul>
-                            <li>
-                                <input type="radio" name="bluetooth" value="on" id="filter-bluetooth-on">
-                                <label for="filter-bluetooth-on">Есть</label>
-                            </li>
-                            <li>
-                                <input type="radio" name="bluetooth" value="off" id="filter-bluetooth-off">
-                                <label for="filter-bluetooth-off">Нет</label>
-                            </li>
-                        </ul>
-                    </fieldset>
-                    <button type="submit">Показать</button>
-                </form>                
-            </aside>
+            <section class="section-filter">
+                <h2 class="visually-hidden">Фильтр товаров</h2>
+                <div class="sorting">
+                    <h2>Сортировка:</h2>
+                    <ul class="sorting-style">
+                        <li>
+                            <label for="price">
+                                <input type="radio" name="content" value="price" id="price" checked>
+                                По цене
+                            </label>
+                        </li>
+                        <li>
+                            <label for="bytype">
+                                <input type="radio" name="content" value="type" id="bytype">
+                                По типу
+                            </label>
+                        </li>
+                        <li>
+                            <label for="popularity">
+                                <input type="radio" name="content" value="popularity" id="popularity">
+                                По популярности
+                            </label>
+                        </li>
+                    <ul class="sorting-direction">
+                        <li>
+                            <label for="sorting-up">
+                                <input type="radio" name="sorting-direction" value="up">
```

Comment:
this can be just buttons for example. why do you think you need inputs here? this is not a form, and no need to make this sortment functional on CSS

Reply:
I had a doubt about radio button but the moment that it can be only two positions: up and down and one is always ON makes me to choose the radio button.
Changed to simple button.